results/README.md

```diff
@@ -13,13 +13,18 @@ API server.
 The API server interface is defined in `./proto/api.proto`, and a reference
 implementation backed by Sqlite will live in `./cmd/api`. A reference
 implementation of the in-cluster watcher will live in `./cmd/watcher`.
+A local Mysql server configuration will be `./config/mysql-*.yaml`.
```

Comment:
We can probably omit this line. It's more of an implementation detail - I think the section below covers this well enough.